numpy/_core/tests/test_custom_dtypes.py

```diff
@@ -298,6 +299,7 @@ def test_flatiter_index(self, index):
         np.testing.assert_array_equal(
             arr.view(np.float64), arr2.view(np.float64))
 
+@pytest.mark.thread_unsafe(reason="_ScaledFloatTestDType setup is thread-unsafe")
```

Comment:
Let's file an issue for this problem that we can link to from these messages. That way if anyone comes along and decides to fix it, they can easily update these tests too.